src/test/robotframework/testsuites/UnitTests/WhatAreCookies.robot

```diff
@@ -1,9 +1,12 @@
 *** Settings ***
 Resource          ../../settings/Settings.robot
+Library    XML
```

Comment:
It seems like we don't use this library here.

Reply:
Probably not, have to check while still making some adjustments to make this pipeline work again.